Source/RunActivity/RunActivity.csproj

```diff
@@ -325,19 +334,18 @@
     <Content Include="Version.txt">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="Viewer3D\WebServices\Web\css\hud.css" />
-    <Content Include="Viewer3D\WebServices\Web\css\sample.css" />
-    <Content Include="Viewer3D\WebServices\Web\css\trackmonitor.css" />
-    <Content Include="Viewer3D\WebServices\Web\HUD.html" />
-    <Content Include="Viewer3D\WebServices\Web\images\or_logo.png" />
-    <Content Include="Viewer3D\WebServices\Web\images\waverley.jpg" />
-    <Content Include="Viewer3D\WebServices\Web\images\zig-zag.jpg" />
-    <Content Include="Viewer3D\WebServices\Web\index - ApiSample.html" />
+    <Content Include="Viewer3D\WebServices\Web\APISample\index.css" />
+    <Content Include="Viewer3D\WebServices\Web\APISample\index.html" />
+    <Content Include="Viewer3D\WebServices\Web\APISample\index.js" />
+    <Content Include="Viewer3D\WebServices\Web\APISample\or_logo.png" />
+    <Content Include="Viewer3D\WebServices\Web\HUD\index.css" />
+    <Content Include="Viewer3D\WebServices\Web\HUD\index.html" />
+    <Content Include="Viewer3D\WebServices\Web\HUD\index.js" />
+    <Content Include="Viewer3D\WebServices\Web\index.css" />
     <Content Include="Viewer3D\WebServices\Web\index.html" />
-    <Content Include="Viewer3D\WebServices\Web\js\ApiSample.js" />
-    <Content Include="Viewer3D\WebServices\Web\js\ApiTrainInfo.js" />
-    <Content Include="Viewer3D\WebServices\Web\js\hud.js" />
-    <Content Include="Viewer3D\WebServices\Web\js\trackmonitor.js" />
+    <Content Include="Viewer3D\WebServices\Web\or_logo.png" />
+    <Content Include="Viewer3D\WebServices\Web\TrackMonitor\index.html" />
+    <Content Include="Viewer3D\WebServices\Web\TrackMonitor\index.js" />
```

Comment:
Are these references needed? I am confused by having these references **and** the `XCOPY` for the same directories, but maybe I'm missing something?

Reply:
@cjakeman